app/controllers/api/v1/users_controller.rb

```diff
@@ -1,4 +1,23 @@
-class Api::V1::UsersController < ApiController
+module Api
+  module V1
+    class UsersController  < ApplicationController
+      skip_before_action :verify_authenticity_token, if: :json_request?
+      skip_before_filter  :verify_authenticity_token
+    def create
+      user = User.new(user_params)
+      if user.save
+        render :json => { :status => :ok, :message => 'Thanks for signing up!'}
```

Comment:
Please make use of http status code, don't keep status inside the response body, we will be making use of http status codes for response codes.